homepage/models.py

```diff
@@ -96,8 +96,20 @@ class Professor_to_Course(models.Model):
     professor_id = models.ForeignKey(Professor, on_delete=models.CASCADE)
     course_id = models.ForeignKey(Course, on_delete=models.CASCADE)
 
+    @staticmethod  # returns a list of Course objects
+    def get_courses_by_professor(professor):

```

Comment:
if I can offer an alternative:
```suggestion
    def get_courses_by_professor(professor):
        prof_to_course_list = Professor_to_Course.objects.filter(professor_id=professor)
        return [arg.course_id for arg in prof_to_course_list]
```

So that you don't have to do a get over Course.objects?

Reply:
But then we’ll get only the id as an integer. Isnt it why wanted to change it in the first place?

Reply:
@ellaml this is from the shell:
![image](https://user-images.githubusercontent.com/62563520/116441678-52867a80-a85a-11eb-91bb-b1dfcf94081e.png)
as you can see this method will return Course objects. It's just that the name of the fk in Professor_to_Course is course_id

Reply:
@kobihk see comment below